commcare_connect/templates/account/account_inactive.html

```diff
@@ -1,11 +1,13 @@
-{% extends "account/base.html" %}
+{% extends "account/base.tailwind.html" %}
```

Comment:
nit: naming. can this just be base.html? We should try to avoid using taliwind in naming. If it can't just be base, we should still avoid the `.tailwind` since two dots like that isn't consistent with other usage.